instrumentation/mongodb-reactive-streams-4.2/src/main/java/com/nr/agent/mongo/MongoUtil.java

```diff
@@ -0,0 +1,108 @@
+/*
+ *
+ *  * Copyright 2021 New Relic Corporation. All rights reserved.
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.nr.agent.mongo;
+
+import com.mongodb.internal.async.SingleResultCallback;
+import com.newrelic.api.agent.DatastoreParameters;
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.Segment;
+import com.newrelic.api.agent.Token;
+
+import java.util.logging.Level;
+
+public class MongoUtil {
+
+    public static final String OP_FIND = "find";
+    public static final String OP_INSERT = "insert";
+    public static final String OP_UPDATE = "update";
+    public static final String OP_AGGREGATE = "aggregate";
+    public static final String OP_REMOVE = "remove";
+    public static final String OP_PARALLEL_SCAN = "parallelCollectionScan";
+    public static final String OP_CREATE_INDEX = "createIndex";
+    public static final String OP_CREATE_INDEXES = "createIndexes";
+    public static final String OP_CREATE_LIST_INDEXES = "createListIndexes";
+    public static final String OP_CREATE_VIEW = "createView";
+    public static final String OP_CREATE_COLLECTION = "createCollection";
+
+    public static final String OP_RENAME_COLLECTION = "renameCollection";
+    public static final String OP_FIND_AND_UPDATE = "findAndUpdate";
+    public static final String OP_FIND_ONE_AND_UPDATE = "findOneAndUpdate";
+
+    public static final String OP_FIND_AND_REPLACE = "findAndReplace";
+    public static final String OP_FIND_ONE_AND_REPLACE = "findOneAndReplace";
+
+    public static final String OP_FIND_AND_DELETE = "findAndDelete";
+    public static final String OP_FIND_ONE_AND_DELETE = "findOneAndDelete";
+
+    public static final String OP_DROP_INDEX = "dropIndex";
+    public static final String OP_DROP_INDEXES = "dropIndexes";
+    public static final String OP_DROP_COLLECTION = "drop";
+    public static final String OP_DROP_DATABASE = "dropDatabase";
+    public static final String OP_DISTINCT = "distinct";
+    public static final String OP_COUNT = "count";
+    public static final String OP_MAP_REDUCE = "mapReduce";
+    public static final String OP_REPLACE = "replace";
+    public static final String OP_REPLACE_ONE = "replaceOne";
+    public static final String OP_LIST_INDEXES = "listIndexes";
+    public static final String OP_LIST_DATABASES = "listDatabases";
+    public static final String OP_LIST_COLLECTIONS = "listCollections";
+    public static final String OP_BULK_WRITE = "bulkWrite";
+    public static final String OP_INSERT_ONE = "insertOne";
+    public static final String OP_INSERT_MANY = "insertMany";
+    public static final String OP_UPDATE_MANY = "updateMany";
+    public static final String OP_UPDATE_ONE = "updateOne";
+
+    public static final String OP_GET_MORE = "getMore";
+    public static final String OP_GROUP = "group";
+    public static final String UNKNOWN = "Unknown";
+
+    // "delete" commands are different from DBCollection.remove
```

Comment:
```suggestion
    /**
    * "delete" commands are different from DBCollection.remove.
    */
```
For consistency with the other comment lines below?

Perhaps also now we are using doc comments:
```suggestion
    /**
    * "delete" commands are different from {@link DBCollection#remove}.
    */
```